Information/Design.md

```diff
@@ -89,3 +89,320 @@
+no parameters or more than name, or id, then the command does nothing, other than print 
+a message stating the context of the syntax issue that is occurring. 
+
+The design for the _user_ command is much bigger in terms of overall scope. Both in Design 
+and implementation. Thus this feature will be held off until the second phase of development. 
+If the backend is change. Thus currently speaking this is a requirement to design hiccup that
+comes as an inevitable consequence form the waterfall process.  
+
+
+examples of _user_ command usage:
+
+`-user Josh`  -- example with _name_ parameter 
+
+`-user 2`     -- example with _id_ parameter 
+
+
+
+------------------------------------------------------------------------------------------
+
+
+######REQ 150 Design:
+
+
+Command name: _-user_ 
+
+Parameters: _name_, _id_ 
+
+**Design of _user_ command**:
+
+
+This command is very similar to **REQ 140** command. Same command name and parameters. 
+Moreover, very similar functionality. The only distinction is that this command does not 
+add new weapons, items, locations, armor or other aspects to the Monster Hunter World 
+database. It simply allows a privileged use to edit currently existing information about
+an item, weapon, location, armour, or other. The program will prompt the user for to enter
+the type of data they wish to edit. If the user inputs an invalid type; other than items,
+weapons, locations or other, that is not apart of the game. The bot will print an 
+appropriate message, and prompt for editing input again. However, if the user again, 
+provides misguided input then the _user_ command exits. However, if the user inputs valid
+input that references a specific type of item in the game. Then the Bot will find that
+item and provide a list of all fields available to edit. This way it creates no ambiguity
+on the users perspective, for what can and cannot be edited in the Monster Hunter 
+World game via the Discord bot.
+
+A privilege user is define by roles in Discord. Each Discord user account has a role 
+on a server that the user's account is a member of. Therefore, if the role is of type 
+"privileged" then the bot will allow that user account to adjust data or edit it in the 
+database of Monster Hunter World video game. 
+
+
+As with **REQ 140**, **REQ 150** is considered more of a bonus feature and therefore is 
+held off until the second phase of development. If the backend is change. Thus, currently 
+speaking this is a requirement to design hiccup that comes as an inevitable consequence 
+form the waterfall process.  
+
+
+
+examples of _user_ command usage:
+
+`-user Josh`  -- example with _name_ parameter 
+
+`-user 2`     -- example with _id_ parameter 
+
+
+
+
+------------------------------------------------------------------------------------------
+
+
+######REQ 160 Design:
+
+
+Command name: _-my victims_ 
+
+Parameters: _name_ 
+
+**Design of _user_ command**:
```

Comment:
This is not the 'user' command

Reply:
LOL, yea I saw this when editing the file just now. I was perplexed at how this occurred as well.